### PR TITLE
docs(bigquery): Add alternative approach to setting credentials

### PR DESCRIPTION
**Pull request by alanyee**
Add alternative approach to setting credentials in bigquery in the comments

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [x] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [x] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Comment by plamut** [comment]
@anguillanneuf Since this is an addition to one of the snippets, could have a quick look at it? Thanks!

File: samples/snippets/authenticate_service_account.py [hunk 1]
```diff
@@ -30,6 +30,11 @@ def main():
         key_path, scopes=["https://www.googleapis.com/auth/cloud-platform"],
     )
 
+    # Alternatively, use service_account.Credentials.from_service_account_info() 
+    # to set credentials from a json string rather than read from a file
```
Comment (tswast) [inline samples/snippets/authenticate_service_account.py:34]: It's not actually a string, though, right? I thought the "info" function takes a parsed value (dictionary).

Comment (alanyee) [reply samples/snippets/authenticate_service_account.py:34]: Right. I will correct that

**Review by tswast — APPROVED** [review]
LGTM. Thanks!

**Comment by tswast** [comment]
Lint is failing:

```
nox > flake8 docs/samples
docs/samples/snippets/authenticate_service_account.py:33:81: W291 trailing whitespace
nox > Command flake8 docs/samples failed with exit code 1
```

File: samples/snippets/authenticate_service_account.py [hunk 2]
```diff
@@ -30,6 +30,11 @@ def main():
         key_path, scopes=["https://www.googleapis.com/auth/cloud-platform"],
     )
 
+    # Alternatively, use service_account.Credentials.from_service_account_info()
```
Comment (plamut) [inline samples/snippets/authenticate_service_account.py:33]:
```suggestion
    # Alternatively, use service_account.Credentials.from_service_account_info()
```

**Pull request merged**